packages/projects-docs/pages/learn/sandboxes/embedding.mdx

```diff
@@ -36,6 +36,10 @@ This will give you HTML code with an iFrame that you can paste into your content
 
 ![Cloud Sandbox Embed](../images/embed-cloud-sandbox.jpg)
 
+<Callout>
+Although microVM embeds are only available for cloud sandboxes, you can still embed a repository by first importing it as a [synced semplate](/learn/sandboxes/synced-templates).
```

Comment:
I find this language a little confusing. It could just be me, but it raises the questions
- are synced templates a type of cloud sandbox?
- is my repo still a repo if I do this, but with embeds now available or is it transformed into a synced template

Reply:
Great observation!